src/main/resources/data-use-v3.json

```diff
@@ -0,0 +1,66 @@
+{
+  "id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Use Schema V3",
+  "version": 2,
+  "type": "object",
+  "dependencies": {
+    "otherRestrictions": {
+      "anyOf": [
+        {"required": ["ethicsApprovalRequired"]},
+        {"required": ["collaboratorRequired"]},
+        {"required": ["geographicalRestrictions"]},
+        {"required": ["other"]}
+      ]
+    }
+  },
+  "properties": {
+    "generalUse": {
+      "type": "boolean"
+    },
+    "diseaseRestrictions": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "uniqueItems": true
+      }
+    },
+    "hmbResearch": {
+      "type": "boolean"
+    },
```

Comment:
One thing we need to figure out here is how to represent the condition that at most one of these three are selected. Let's circle back with Jonathan on this question and review what we're doing on both the dataset and dar UI forms. This might be helpful: https://json-schema.org/understanding-json-schema/reference/combining.html?highlight=oneof

Reply:
Sorry, that comment didn't capture the three: `generalUse`, `diseaseRestrictions`, or `hmbResearch`.

Reply:
Do we need to encode this before merging?